CHANGES.rst

```diff
@@ -18,6 +18,15 @@
 Changes
 =======
 
+`0.3.8 <https://github.com/SwissDataScienceCenter/calamus/compare/v0.3.7...v0.3.8>`__ (2021-04-26)
+--------------------------------------------------------------------------------------------------
+
+Fixes
+~~~~~
+
+- Fixed an issue when deserializing raw fields containing JSON-LD.
+  (`PR #58 <https://github.com/SwissDataScienceCenter/calamus/pull/58>`__)
```

Comment:
To me this looks to be more of a feature than a fix; a new RawJsonLD is added. Should we change this to a feature?

Reply:
sure thing! lemme forcepush the change :)